config/routes.rb

```diff
@@ -2,7 +2,8 @@
   ActiveAdmin.routes(self)
   root :to => 'home#index'
   get '/error' => 'home#error'
-  get '/me'    => 'home#me'
+  get '/me'    => 'profile#show'
```

Comment:
ah yeah sorry about vagueness -- so in here, what I meant by regular is that we'd have `resources :profiles, only: [:edit, :update]` per the Rails default way of defining routes http://guides.rubyonrails.org/routing.html